### PR TITLE
aise hi curious q

### PR DESCRIPTION
**Pull request by ShashankSubham**

### CODE REVIEW

**Comment by SakshiShreya** [comment]
Hi! I think this PR looks good. You can merge it.

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+kya baat h muton pe aa gyi chicken se
```
Comment (SakshiShreya) [inline file2.txt]: Aisa to nahi h :laughing:

Comment (ShashankSubham) [reply file2.txt]: koi nhi khane se mtlb h

Comment (SakshiShreya) [reply file2.txt]: Acha ok. accept kar rahi hu

**Pull request merged**